src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyTimestampTest.java

```diff
@@ -0,0 +1,111 @@
+package com.microsoft.sqlserver.jdbc.bulkCopy;

```

Comment:
this is only 1 test doesn't seem necessary to add a new file, please just add the test to BulkCopyAllTypesTest.java